README.md

```diff
@@ -556,40 +577,41 @@ By default, this module will create a resource group and the name of the resourc
 
 > **Note:** *If you are using an existing resource group, then this module uses the same resource group location to create all resources in this module.*
 
-## Management Operations Logging
+## Azure Network DDoS Protection Plan
 
-This module enables diagnostic settings for Azure resources that emit platform logs. The diagnostic settings are configured to send platform logs to a Log Analytics workspace. The Log Analytics workspace is created in the same resource group as the hub virtual network. The Log Analytics workspace is configured to retain data for 30 days. The Log Analytics workspace is configured to use the Standard pricing tier.
+By default, this module will not create a DDoS Protection Plan. You can enable/disable it by appending an argument `create_ddos_plan` located in `variables.vnet.tf`. If you want to enable a DDoS plan using this module, set argument `create_ddos_plan = true`
 
-As part of SCCA compliance, Management Operations Logging is enabled.
+## Azure Network Network Watcher
 
-Diagnostic settings are controlled trough Policy. Policy will create a policy assignment to enable diagnostic settings for all resources in the resource group.
+This module handles the lookup of Network Watcher resource. It will use network watcher, flow logs and traffic analytics for all the subnets in the Virtual Network.
 
-The following Azure resources can be configured to send platform logs to the Log Analytics workspace:
+> **Note:** *Log Analytics workspace is required for NSG Flow Logs and Traffic Analytics. If you want to enable NSG Flow Logs and Traffic Analytics, you must create a Log Analytics workspace and provide the workspace name set argument `log_analytics_workspace_name` and rg set argument `log_analytics_workspace_resource_group_name`*
 
-* Azure Firewall
-* Azure Storage
-* Azure Key Vault
-* Azure Application Gateway
-* Azure Load Balancer
-* Azure Network Security Group
-* Azure Virtual Network Gateway
-* Azure Virtual Network
+## Enable Force Tunneling for the Firewall
 
-> **NOTE:**  Please review the [Mission Enclave Policy Starter](https://github.com/azurenoops/ref-scca-enclave-policy-starter) reference implementation for more information.
+By default, this module will not create a force tunnel on the firewall. You can enable/disable it by appending an argument `enable_force_tunneling` located in `variables.fw.tf` If you want to enable a DDoS plan using this module, set argument `enable_force_tunneling = true`. Enabling this feature will ensure that the firewall is the default route for all the T0 through T3 Network routes.
```

Comment:
The third sentence says "If you want to enable a DDoS Plan..."

Reply:
fixed